### PR TITLE
Beast latest version using foss-2016a

### PR DESCRIPTION
**Pull request by ysagon**

### CODE REVIEW

**Comment by ysagon** [comment]
depends on #2863

File: easybuild/easyconfigs/b/Beast/Beast-2.4.0-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+source_urls = ['https://github.com/CompEvol/beast2/releases/download/v%(version)s/']
+sources = ['BEAST.v%(version)s.Linux.tgz']
+
+
+dependencies = [
+    # this is not mandatory but beagle-lib is recommended by developers
+    # beagle-lib will also load the required java dependency
+    # if you remove this you should add the java dependency
+    ('beagle-lib', '2_1_2', '', ('foss', '2016a')),
```
Comment (boegel) [inline easybuild/easyconfigs/b/Beast/Beast-2.4.0-foss-2016a.eb]: please use `2.1.2` as version number for `beagle-lib` (see also remarks in #2863)

There's no need to explicitly specify the toolchain if is matches that of the parent, so stick to `('beagle-lib', '2.1.2')`

File: easybuild/easyconfigs/b/Beast/Beast-2.4.0-foss-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ but is also a framework for testing evolutionary hypotheses without conditioning on a single 
+ tree topology. BEAST uses MCMC to average over tree space, so that each tree is weighted 
+ proportional to its posterior probability. """
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/CompEvol/beast2/releases/download/v%(version)s/']
+sources = ['BEAST.v%(version)s.Linux.tgz']
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/b/Beast/Beast-2.4.0-foss-2016a.eb]: style nitpicking: please drop extra empty line

**Comment by boegel** [comment]
Jenkins: ok to test

(tests will fail while #2863 isn't merged yet, but we can retrigger them once it is)

**Comment by boegel** [comment]
lgtm (but on hold until #2863 gets merged)

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2613.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6c3060132ed6fe0ce39a90830509cab7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2038.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/935ffd59b3528287170d2ffe54aad40a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ysagon!

**Pull request merged**